src/tensor_ops/boolean/cuda_kernels.rs

```diff
@@ -0,0 +1,106 @@
+use super::BooleanKernel;
+use crate::prelude::{cuda::CudaArray, *};
+use cudarc::prelude::*;
+
+use std::sync::Arc;
+
+const MODULE_NAME: &str = "/boolean.ptx";
```

Comment:
nitpick for consistency
```suggestion
const MODULE_NAME: &str = "boolean";
```